apps/ui/components/reservation/ReservationCard.tsx

```diff
@@ -152,24 +152,19 @@ const Image = styled.img`
 function StatusTags({
   state,
   orderStatus,
-  statusType = "desktop",
 }: {
   state: ReservationStateChoice;
   orderStatus: OrderStatus | null;
-  statusType: "desktop" | "mobile";
```

Comment:
Removing this is probably gonna break tests, because there is still a duplicated JSX (one for mobile, one for desktop). Either keep it or refactor the `JustForMobile` and `JustForDesktop`

Reply:
This will be gone with the new Cards anyway...

Reply:
To clarify my point: it feels unnecessary to separately start refactoring away `JustForMobile` and `JustForDesktop` as there's an entirely new thing in another branch coming up which will replace those anyway.

Reply:
..but I guess removing them then is just as much ok, IMHO :)